### ISSUE TITLE
Rule to test for focusable elements with aria-hidden="true" or role="presentation"

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#6]
If you have a naturally tab-focusable element (that is not display:none or visibility:hidden) such as a button with aria-hidden="true", screen reader users will hear "blank" when they land on the element. We need a rule to test for these conditions

http://www.w3.org/TR/aria-in-html/#fourth-rule-of-aria-use

**Comment by dsturley** [comment]
Should probably specify this should be "tabbable" elements, not just focusable.  A good example of this distinction is the common ARIA menu pattern:

```html
<ul role="menu">
   <li role="menuitem" tabindex="0">
      <a href="https://github.com" role="presentation" tabindex="-1">GitHub</a>
   </li>
   <li role="menuitem" tabindex="-1">
      <a href="https://bitbucket.org" role="presentation" tabindex="-1">BitBucket</a>
   </li>
</ul>
```

**Comment by dsturley** [comment]
Also `summary` element for HTML5

**Comment by WilcoFiers** [comment]
This rule was added in axe-core 3.3.
